rpc/js/src/BaseChannel.ts

```diff
@@ -1,4 +1,4 @@
-import type { ProtobufMessage } from '@improbable-eng/grpc-web/dist/typings/message';
+import { Message } from '@bufbuild/protobuf';
```

Comment:
Import only used as type

```suggestion
import type { Message } from '@bufbuild/protobuf';
```

Reply:
shouldn't all these kinds of things be picked up by a linter?

Reply:
Yup, see https://github.com/viamrobotics/goutils/pull/354#discussion_r1771674359